src/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To Do List</title>
+  </head>
+  <body>
+    <h1 class="bg-title">ToDo</h1>
+    <main id="todo-wrapper">
+      <div class="todo-header">
+        <h4>Today's To Do</h4>
+        <a href="#">
+          <svg
+            class="icon"
+            fill="none"
+            stroke="currentColor"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              stroke-width="2"
+              d="M4 4v5h.582m15.356 2A8.001 8.001 0 004.582 9m0 0H9m11 11v-5h-.581m0 0a8.003 8.003 0 01-15.357-2m15.357 2H15"
+            ></path>
+          </svg>
+        </a>
+      </div>
+      <form id="todo-form">
+        <input
```

Comment:
Optional, it would be cleaner if you had a label with the id of the input type.

Reply:
I agree with your that lacking label for input is an issue. but the design given to us didn't have label. So that is a design problem.

After we finish our work on this project, I will redesign this project and that time I try to take care of such things.

Thanks